### PR TITLE
Add instructions for submission upload using cli on Submit page

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR adds -

- [x] Instructions to make a submission using EvalAI-CLI for large submission upload

Screen Shot 2020-11-11 at 9 12 26 PM

### CODE REVIEW

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
Can we also add instruction to make the submission `private` while submitting using CLI for both large files and code upload challenge.

**Comment by Ram81** [comment]
Screen Shot 2020-11-12 at 11 56 43 AM
Screen Shot 2020-11-12 at 12 21 08 PM

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
Can we please add what `--private` does in docker based challenge. Please add it as point 4 in submit instructions.

**Comment by Ram81** [comment]
@RishabhJain2018 
Screen Shot 2020-11-14 at 12 00 50 PM

File: frontend/src/views/web/challenge/submission.html [hunk 1]
```diff
@@ -61,6 +61,14 @@ <h5 class="w-300">
                             <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard."></i>
                         </button>
                     </li>
+                    <li>Make private submission<br>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/challenge/submission.html:64]: Can we please change it to -- Use `--private` or `--public` flag in the submission command to make the submission private or public respectively.

File: frontend/src/views/web/challenge/submission.html [hunk 2]
```diff
@@ -61,6 +61,14 @@ <h5 class="w-300">
...
+                        <code>$ evalai push &lt;image&gt;:&lt;tag&gt; --phase &lt;phase_name&gt; --private</code>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/challenge/submission.html:65]: Please remove the command as it is already added for each phase.

File: frontend/src/views/web/challenge/submission.html [hunk 3]
```diff
@@ -351,22 +364,69 @@ <h5 class="w-300">Make Submission</h5>
...
+                                        3. Make submission<br>
+                                        <code>$ evalai challenge {{challenge.challengeId}} phase {{challenge.phaseId}} submit &lt;submission_file_path&gt; --large</code>
+                                        <button class="btn ev-btn-dark waves-effect waves-dark grad-btn code-btn"
+                                            ng-click="challenge.showConfirmation('Command copied to clipboard')" ngclipboard
+                                            data-clipboard-text="evalai challenge {{challenge.challengeId}} phase {{challenge.phaseId}} submit &lt;submission_file_path&gt; --large">
+                                            <i class="fa fa-clipboard" aria-hidden="true" alt="Copy to clipboard."></i>
+                                        </button>
+                                    </li>
+                                    <li>
+                                        4. Make private submission<br>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/challenge/submission.html:416]: Use --private or --public flag in the submission command to make the submission private or public respectively.

File: frontend/src/views/web/challenge/submission.html [hunk 4]
```diff
@@ -351,22 +364,69 @@ <h5 class="w-300">Make Submission</h5>
...
+                                        <code>$ evalai challenge {{challenge.challengeId}} phase {{challenge.phaseId}} submit &lt;submission_file_path&gt; --large --private</code>
```
Comment (RishabhJain2018) [inline frontend/src/views/web/challenge/submission.html:417]: Please remove this command.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

**Comment by Ram81** [comment]
@RishabhJain2018 
Screen Shot 2020-11-14 at 3 02 54 PM
Screen Shot 2020-11-14 at 3 03 32 PM

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**